opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1169,6 +1173,7 @@ namespace Opm {
             if (terminal_output_ && (network_update_iteration == iteration_to_relax) ) {
                 local_deferredLogger.info(" we begin using relaxed tolerance for network update now after " + std::to_string(iteration_to_relax) + " iterations ");
             }
+           
```

Comment:
This is an extra line not related to this PR.

Reply:
Done